canonical/charmhub.io
# Move publicise to react app

## Done
Moved the publisher publicise section into the publisher React application

## How to QA
- Go to https://charmhub-io-1810.demos.haus/<CHARM_NAME>/publicise
- Check that the "Charmhub buttons" section is selected and compare to production (**Note**: The markdown section is different because it is wrong in production)
- Go to https://charmhub-io-1810.demos.haus/<CHARM_NAME>/publicise/badges and compare to production
- Go to https://charmhub-io-1810.demos.haus/<CHARM_NAME>/publicise/cards and compare to production

## Issue / Card
Fixes https://warthogs.atlassian.net/browse/WD-10531
